src/platforms/flutter/troubleshooting.mdx

```diff
@@ -10,7 +10,7 @@ If you need help solving issues with Sentry's Flutter SDK, you can read the edge
 
 ## Known Limitations
 
-- If you enable the `split-debug-info` feature, you must upload the Debug Symbols manually.
+- If you enable the `split-debug-info` and `obfuscate` feature, you must upload [Debug Symbols](/platforms/flutter/upload-debug/).
```

Comment:
Are these one feature or two different ones? If two features, then it needs a small edit.

```suggestion
- If you enable the `split-debug-info` and `obfuscate` features, you must upload [debug symbols](/platforms/flutter/upload-debug/).
```